src/style.css

```diff
@@ -1,3 +1,95 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
+  line-height: 14px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  background-color: gray;
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  background-color: white;
+  display: flex;
+  max-width: auto;
+  max-height: auto;
```

Comment:
Avoid fixing container sizes using `max-width: auto;` and `max-height: auto;`. Let the content size dictate the container size to prevent overflow or accidental scrollbars.